src/components/sponsorship/SponsorMarketplace.tsx

```
@@ -74,11 +76,16 @@ export const SponsorMarketplace: React.FC<SponsorMarketplaceProps> = ({
                     <p className="text-xl font-black text-indigo-600 dark:text-indigo-400">
                         {formatCurrency(requestedAmount)}
                     </p>
-                </div>
+</div>
             </div>
 
-            {matches.length === 0 ? (
-                <div className="text-center py-16 bg-gray-50 dark:bg-gray-800/50 rounded-xl border-2 border-dashed border-gray-300 dark:border-gray-700">
+            {clubId && (
```

ESLint

[error] Replace `············` with `······`

(prettier/prettier)